examples/solvers/rosenbrock_minimization.py

```diff
@@ -38,24 +39,35 @@
 x = space.zero()
 odl.solvers.steepest_descent(f, x, line_search=line_search,
                              callback=callback)
+legend_artists = [callback.ax.collections[-1], ]
```

Comment:
This is an OK change for now, but I'll see if we can add this functionality to `CallbackShowConvergence` in the long term, see #1121